airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab.tsx

```diff
@@ -95,11 +95,11 @@ const NormalizationCard: React.FC<{
 export const ConnectionTransformationTab: React.FC = () => {
   const { connection, updateConnection } = useConnectionEditService();
   const { mode } = useConnectionFormService();
-  const definition = useGetDestinationDefinitionSpecification(connection.destination.destinationDefinitionId);
+  const definition = useDestinationDefinition(connection.destination.destinationDefinitionId);
   const workspace = useCurrentWorkspace();
 
   useTrackPage(PageTrackingCodes.CONNECTIONS_ITEM_TRANSFORMATION);
-  const { supportsNormalization } = definition;
+  const supportsNormalization = Boolean(definition.normalizationConfig);
```

Comment:
(nit) This is the conversion I was a little worried about above - `Boolean(null) == false`, but `Boolean({}) == true`.  If we were using `definition.normalizationConfig.supported` it would be a little safer

Reply:
I implemented this suggestion. Thank you.